Setup.sh

```diff
@@ -67,6 +67,10 @@ markStartOfBlock "Setup variables"
     BINARIES_DIR="$(dirname "$0")/SpatialGDK/Binaries/ThirdParty/Improbable"
     SCHEMA_COPY_DIR="$(dirname "$0")/../../../spatial/schema/unreal/gdk"
     SCHEMA_STD_COPY_DIR="$(dirname "$0")/../../../spatial/build/dependencies/schema/standard_library"
+	DOMAIN_ENVIRONMENT_VAR=
```

Comment:
Nit: Use spaces in scripts (just to stay consistent)
```suggestion
    DOMAIN_ENVIRONMENT_VAR=
```